src/main/java/com/sprints/SprintsApiApplication.java

```diff
@@ -21,10 +17,6 @@ public static void main(String[] args) {
 	
 	@Configuration
 	public static class WebMvcConfig implements WebMvcConfigurer  {	
-		
-		public void addInterceptors(InterceptorRegistry registry) {
```

Comment:
Why are you deleting this configuration ?

Reply:
Since we no longer validate that invalid query parameters will return an exception, the interceptors configuration was unnecessary